build/source/driver/multi_driver.f90

```diff
@@ -325,10 +481,10 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
 allocate(upArea(nHRU),stat=err); call handle_err(err,'problem allocating space for upArea')
 
 ! identify the total basin area (m2)
-totalArea => bvar_data%var(iLookBVAR%basin__totalArea)%dat(1)
+totalArea = bvarStruct%var(iLookBVAR%basin__totalArea)%dat(1)
```

Comment:
This is an error. The variable `totalArea` actually needs to be associated with (or point to) the data structure, so that the information in the data structure is updated and included in the output files.